lib/gui/lvgl/Kconfig

```diff
@@ -21,15 +24,17 @@ config APP_LINK_WITH_LVGL
 	  disabled if the include paths for LVGL are causing aliasing
 	  issues for 'app'.
 
+config LVGL_USE_FILESYSTEM
```

Comment:
I think `LV_` prefix should be used everywhere for consistency, maybe use `LV_Z` for Zephyr specific options?

Reply:
Done, changed to LV_Z_.